### ISSUE TITLE
WordPress.WhiteSpace.ControlStructureSpacing introduces a bunch of false positives

### ISSUE THREAD

**Issue by danielbachhuber** [WordPress-Coding-Standards/WordPress-Coding-Standards#425]
Here's the build failure: https://travis-ci.org/fusioneng/Shortcake/builds/77176720

Fails on these lines:
* https://github.com/fusioneng/Shortcake/blob/3df0e8a88ccc396a651a8026abb4cde481123c1b/inc/fields/class-field-attachment.php#L114
* https://github.com/fusioneng/Shortcake/blob/3df0e8a88ccc396a651a8026abb4cde481123c1b/inc/fields/class-field-color.php#L121

How are those failures supposed to be resolved?

**Comment by JDGrimes** [comment]
Strangely, I'm unable to reproduce this. However, I *think* that it is warning about the blank line between the last method and the class closing brace.

```php
class Example {

    function method() {
    }
     // < - It doesn't like this
}
```

**Comment by danielbachhuber** [comment]
> However, I think that it is warning about the blank line between the last method and the class closing brace.

Ah, that's it. Is the error correct though?

**Comment by JDGrimes** [comment]
Yes. I'm not sure if it is really specified by WordPress's standards, but it is something that WPCS checks. If you want you can disable just that error.

**Comment by JDGrimes** [comment]
Just FYI, you can disable just this error without disabling the whole sniff:

```xml
 	<exclude name="WordPress.WhiteSpace.ControlStructureSpacing.BlankLineAfterEnd" /> 
```
